sites/example-project/src/components/viz/DataTable.svelte

```diff
@@ -319,7 +332,34 @@
                   class:row-lines={rowLines}
                   style="
                       text-align: {column.align};
-                  ">{formatValue(row[column.id], columnSummary.filter(d => d.id === column.id)[0].format)}</td>
+                      height: {column.height};
+                      width: {column.width};
+                  ">
+                  {#if column.contentType === "image"}
+                    <img 
+                    src={row[column.id]} 
+                    alt={row[column.id]} 
+                    style="
```

Comment:
So this is always going to use the url as the alt tag for the image.

Do we want to either:
- process the regex from the URL, perhaps using anything after the last slash
- allow users to override the alt tag with another prop?

Reply:
Good call - I've just added a prop for `alt`.

Regex seems like a pretty good idea for that. If we do that, should we include the file extension?

Reply:
I've added regex to take the filename after the last slash and strip off the extension